src/openrct2/object/Object.h

```diff
@@ -191,7 +193,7 @@ struct ObjectEntryDescriptor
     // JSON
     ObjectType Type{};
     std::string Identifier;
-    std::string Version;
+    std::string version;
```

Comment:
Our fields use TitleCase.

Reply:
This was done because of the naming conflicts with the `Version` type. The Version type could be given a different name in a future PR but I'm not coming up with a good one right now.

Reply:
You can differentiate between them by prefixing the type with `::`, like we did in some other places. Also, the tuple definition should be put into that other PR anyway, considering it’s now unused in this PR.